tools/build.sh

```diff
@@ -40,20 +60,28 @@ replace_icon_sprite_url() {
     OLD_PATH="\.\.\/\.\.\/\.\.\/node_modules\/@nimiq\/style\/nimiq-style.icons.svg"
     NEW_PATH="\/assets\/nimiq-style.icons.svg"
 
-    sed -i -e "s/$OLD_PATH/$NEW_PATH/g" $1
+    inplace_sed "s/$OLD_PATH/$NEW_PATH/g" $1
 }
 
 # replace font url in file $1
 replace_font_url() {
     OLD_PATH="(\.\.\/)*assets\/fonts"
     NEW_PATH="\/assets\/fonts"
 
-    sed -i -r -e "s/$OLD_PATH/$NEW_PATH/g" $1
+    inplace_sed -E "s/$OLD_PATH/$NEW_PATH/g" $1
+}
+
+# Replace xxd -r -p with a nice bash function
+function hex_to_binary() {
+  for (( i=0; i<${#1}; i+=2 ))
+  do
```

Comment:
```suggestion
  for (( i=0; i<${#1}; i+=2 )); do
```

Reply:
Yep, you're right. I also noticed that, but didn't thought it's important enough to deserve dedicated attention given the other potential weak spots of the build script... :)